scilpy/tractograms/tests/test_dps_and_dpp_management.py

```diff
@@ -27,45 +30,112 @@ def _get_small_sft():
     return fake_sft
 
 
-def test_add_data_as_color_dpp():
-    lut = get_lookup_table('viridis')
+def test_get_data_as_arraysequence_dpp():
+    fake_sft = _get_small_sft()
+
+    some_data = np.asarray([2, 20, 200, 0.1, 0.3, 22, 5])
+
+    # Test 1: One value per point.
+    array_seq = get_data_as_arraysequence(some_data, fake_sft)
+
+    assert fake_sft._get_point_count() == array_seq.total_nb_rows
+
+
+def test_get_data_as_arraysequence_dps():
+    fake_sft = _get_small_sft()
+
+    some_data = np.asarray([2, 20])
+
+    # Test: One value per streamline.
+    array_seq = get_data_as_arraysequence(some_data, fake_sft)
+    assert fake_sft._get_streamline_count() == array_seq.total_nb_rows
+
+
+def test_get_data_as_arraysequence_dps_2D():
+    fake_sft = _get_small_sft()
+
+    some_data = np.asarray([[2], [20]])
+
+    # Test: One value per streamline.
+    array_seq = get_data_as_arraysequence(some_data, fake_sft)
+    assert fake_sft._get_streamline_count() == array_seq.total_nb_rows
+
+
+def test_get_data_as_arraysequence_error():
+    fake_sft = _get_small_sft()
+
+    some_data = np.asarray([2, 20, 200, 0.1])
+
+    # Test: Too many values per streamline, not enough per point.
+    with pytest.raises(ValueError):
+        _ = get_data_as_arraysequence(some_data, fake_sft)
+
 
-    # Important. cmap(1) != cmap(1.0)
-    lowest_color = np.asarray(lut(0.0)[0:3]) * 255
-    highest_color = np.asarray(lut(1.0)[0:3]) * 255
+def test_add_data_as_dpp_1_per_point():
 
     fake_sft = _get_small_sft()
+    cmap = get_lookup_table('jet')
 
     # Not testing the clipping options. Will be tested through viz.utils tests
 
     # Test 1: One value per point.
     # Lowest cmap color should be first point of second streamline.
```

Comment:
Can you remove this sentence (`Lowest cmap color should be first point of second streamline.`). It sound like we will test the values, but we don't anymore, because the method "add_data_as_color_dpp" does not do any clipping / changing the range anymore.

It is possible to add these tests that I had prepared into the tests of clip_and_normalize_data_for_cmap?